examples/35_nesting/src/entries.tsx

```diff
@@ -32,22 +33,43 @@ export default defineEntries({
       );
     }
   },
-  getBuildConfig: async () => [
-    {
-      pathSpec: [],
-      entries: [
-        { rscPath: '' },
-        { rscPath: 'InnerApp=1', skipPrefetch: true },
-        { rscPath: 'InnerApp=2', skipPrefetch: true },
-        { rscPath: 'InnerApp=3', skipPrefetch: true },
-        { rscPath: 'InnerApp=4', skipPrefetch: true },
-        { rscPath: 'InnerApp=5', skipPrefetch: true },
-      ],
-    },
-    {
-      pathSpec: [{ type: 'literal', name: '/no-ssr' }],
-      entries: [{ rscPath: 'AppWithoutSsr' }],
-      isStatic: true,
-    },
-  ],
+  handleBuild: ({
+    renderRsc,
+    rscPath2pathname,
+    unstable_generatePrefetchCode,
+  }) =>
+    createAsyncIterable(async () => {
+      const moduleIds = new Set<string>();
+      const generateHtmlHead = () =>
+        `<script type="module" async>${unstable_generatePrefetchCode(
+          [''],
+          moduleIds,
+        )}</script>`;
+      const tasks = [
+        async () => ({
+          type: 'file' as const,
+          pathname: rscPath2pathname(''),
+          body: renderRsc(
+            { App: <App name="Waku" />, InnerApp: <InnerApp count={0} /> },
+            { moduleIdCallback: (id) => moduleIds.add(id) },
+          ),
+        }),
+        ...[1, 2, 3, 4, 5].map((count) => async () => ({
+          type: 'file' as const,
+          pathname: rscPath2pathname(`InnerApp=${count}`),
+          body: renderRsc({ App: <App name="Waku" /> }),
+        })),
+        async () => ({
+          type: 'defaultHtml' as const,
+          pathname: '/',
+          htemlHead: generateHtmlHead(),
+        }),
+        async () => ({
+          type: 'defaultHtml' as const,
+          pathname: '/no-ssr',
+          htemlHead: generateHtmlHead(),
```

Comment:
@dai-shi I think htemlHead might be a typo?

Reply:
Oh, no. Yeah, thanks for pointing it out.